implementations/rust/ockam/ockam_api/src/control_api/backend/inlet.rs

```diff
@@ -47,24 +48,29 @@ impl HttpControlNodeApiBackend {
                 warn!("Invalid method: {method}");
                 ControlApiHttpResponse::invalid_method(
                     method,
-                    vec!["PUT", "GET", "PATCH", "DELETE"],
+                    vec!["POST", "GET", "PATCH", "DELETE"],
                 )
             }
         }
     }
 }
 
 #[utoipa::path(
-    put,
+    post,
     operation_id = "create_tcp_inlet",
     summary = "Create a new TCP Inlet",
-    path = "/{node}/tcp-inlet",
-    tags = ["portal", "tcp-inlet"],
+    description =
+"Create a TCP Inlet, the main parameters are the destination `to`, and the bind address `from`.
+You can also choose to the listen with a valid TLS certificate, restrict access to the Inlet with
+`authorized` and `allow`, and select a specialized Portals with `kind`.
+The creation will be asynchronous and the initial status will be `down`.",
```

Comment:
`down` or `DOWN`?

Reply:
Everything uses kebab-case, enums included, so it's rendered as `down` and `up`